aiohttp/web_urldispatcher.py

```diff
@@ -230,8 +231,24 @@ def set_current_app(self, app):
     def freeze(self):
         self._frozen = True
 
+    @property
+    def variable_maps(self):
```

Comment:
Maybe let's use just `maps` name?
`maps` is shorted, in documentation we could refer to `ChainMap.maps`. `maps` is mentally more familiar to people if they know about `ChainMap` already.

Reply:
This is also related to the resolution order of the chain map and the build order of it is reversed.
Also I wanted to make it read-only since `ChainMap`'s `maps` property is mutable.